apps/rehash.c

```diff
@@ -51,6 +51,24 @@
 # endif
 # define MAX_COLLISIONS  256
 
+# if defined(OPENSSL_SYS_VXWORKS)
+/* VxWorks has no symbolic links */
+
```

Comment:
delete this blank line to "bind" the comment more closely?
or turn it into a multi-line comment with three lines /*, text, */ each on separate ines.